_posts/2016-07-03-handling-big-projects.markdown

```diff
@@ -0,0 +1,123 @@
+---
+layout: post
+title: "Handling Large OSS Projects"
+date: 2016-07-03 12:00
+author: orta
+categories: [mobile, oss, culture]
+---
+
+I maintain big OSS projects: ranging from a third-party [dependency manager][cocoapods] used in most iOS apps, to the most popular Objective-C [testing framework][specta] and the most popular Swift [networking client][moya]. I've been doing this for years.
```

Comment:
Changing "I maintain" to "I help maintain" may help in sounding less congratulatory. I think Moya is "one of" the most popular networking clients, we're still 14k stars behind Alamofire.